package.json

```diff
@@ -43,6 +43,7 @@
     "babel-jest": "^26.0.1",
     "babel-loader": "^8.1.0",
     "core-js": "^3.6.5",
+    "crypto": "^1.0.1",
```

Comment:
Is this right? `crypto` is a core node API, so likely this shouldn't be installed as a dependency https://www.npmjs.com/package/crypto

Reply:
Dev dependency only - it's used to generate some random bytes in tests, could be replaced if necessary

Reply:
as in, there's likely no need to include it as a dependency at all, because it's built into node core.

See https://www.npmjs.com/package/crypto

![image](https://user-images.githubusercontent.com/43438/82933162-85900200-9f57-11ea-9019-cb1d9d083786.png)